contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -4610,6 +4613,35 @@ exec_stmt_execsql(PLtsql_execstate *estate,
 	if (stmt->original_query)
 		original_query_string = stmt->original_query;
 
+	/*
+	 * In the query if there are multiple white spaces after dbo then
+	 * replace the multiple spaces with 1 single white space
+	 */
+	if(strcasestr(expr->query, "dbo  "))
```

Comment:
this function is called for each query in a function/procedure/block. Pleas make sure that no significant overhead is added due to these checks. Ideally, we want to club these checks someplace where we are already processing the query.

Reply:
Can you check the possibility of rewriting this in "pre_parse_analyze_hook"?